src/utils/index.ts

```diff
@@ -1,196 +1,64 @@
-  return name === config.DISCORD_GUILD_MASTER_USERNAME;
-};
-
-export const isBroadcaster = name => {
-  return name === config.BROADCASTER_USERNAME;
-};
-
-type TwitchProfile = {
-  id: string;
-  login: string;
-  display_name: string;
-  type: string;
-  broadcaster_type: string;
-  description: string;
-  profile_image_url: string;
-  offline_image_url: string;
-  view_count: number;
-};
-
-let twitchProfileCache = new Map<string, TwitchProfile>();
-
-function addProfileToCache(
-  username: string,
-  profile: TwitchProfile
-): TwitchProfile {
-  if (twitchProfileCache.has(username)) {
-    return profile;
-  }
-  twitchProfileCache.set(username, profile);
-  setTimeout(() => twitchProfileCache.delete(username), 1000 * 60 * 60 * 24);
-  return profile;
-}
-
-const getTwitchProfile = async (username: string): Promise<TwitchProfile> => {
-  if (twitchProfileCache.has(username)) {
-    return twitchProfileCache.get(username);
-  }
-
-  try {
-    const {
-      data: [profile = null]
-    } = await callTwitchApi("https://api.twitch.tv/helix/users", {
-      qs: { login: username.toLowerCase() }
-    });
-    return addProfileToCache(username, profile);
-  } catch (e) {
-    BeastieLogger.warn(`Failed to get twitch user ${username}: ${e}`);
-  }
-
-  return null;
-};
-
-export const getTwitchId = async (username: string): Promise<string> => {
-  let user = await getTwitchProfile(username);
-  return user?.id;
-};
-
-export const getBroadcasterId = async (): Promise<string> => {
-  return getTwitchId(config.BROADCASTER_USERNAME);
-};
-
-export const getBroadcasterDisplayName = async () => {
-  const userArray = await getTwitchProfile(config.BROADCASTER_USERNAME);
-  return userArray?.display_name;
-};
-
-const getBroadcasterStream = async broadcasterID => {
-  return callTwitchApi(
-    `https://api.twitch.tv/helix/streams?first=1&user_id=${broadcasterID}`,
-    {}
-  );
-};
-
-export const initStream = async () => {
-  const broadcasterID = await getBroadcasterId();
-  const stream = await getBroadcasterStream(broadcasterID);
-
-  const live = stream.data[0] && stream.data[0].type === "live";
-  const id = stream.data[0] && stream.data[0].id;
-
-  return { live, id };
-};
-
-type ChattersData = {
-  broadcaster?: string[];
-  vips?: string[];
-  moderators: string[];
-  staff: string[];
-  admins: string[];
-  global_mods: string[];
-  viewers: string[];
-};
-
-async function getTwitchProfiles(usernames: string[]) {
-  let twitchProfiles: TwitchProfile[] = [];
-
-  usernames = usernames.filter((username: string): boolean => {
-    if (!twitchProfileCache.has(username)) {
-      return true;
-    }
-
-    twitchProfiles.push(twitchProfileCache.get(username));
-    return false;
+import { TwitchProfile } from "../services/twitch/TwitchAPI/apiTypes";
+import {
+  getTwitchChatters,
+  getTwitchProfile
+} from "../services/twitch/TwitchAPI";
+import { default as fetch, RequestInfo } from "node-fetch";
+
+export async function PerformGetRequest<T>(
```

Comment:
I think that because `PerformGetRequest` is not a class or constructor, the first character should be lowercase.

No patch suggestion given, because it has references in multiple files. :)